packages/upload-api/src/types/delegations.ts

```diff
@@ -5,6 +5,16 @@ interface ByAudience {
 }
 export type Query = ByAudience
 
+export interface RevocationMeta {
+  context: Ucanto.Link
```

Comment:
Perhaps we should call this a `scope` instead, as in UCAN has been revoked in this scope.

```suggestion
  scope: Ucanto.Link
```